services/ui-src/src/components/reports/DrawerReportPage.tsx

```diff
@@ -121,8 +147,15 @@ export const DrawerReportPage = ({ route, validateOnRender }: Props) => {
     onClose();
   };
 
+  const openRowDrawer = (entity: EntityShape) => {
+    setSelectedEntity(entity);
+    onOpen();
+  };
+
   const entityRows = (entities: EntityShape[]) => {
-    const disabled = reportingOnIlos && !hasIlos;
+    // TODO: CLEAN UP THIS THING
+    let disableIlos = reportingOnIlos && !hasIlos && true;
```

Comment:
`&& true` won't change anything here if any of the other values are false. Do we want `|| true`?